ksqldb-streams/src/main/java/io/confluent/ksql/execution/streams/timestamp/LoggingTimestampExtractor.java

```diff
@@ -70,7 +65,8 @@ public long extract(final GenericRow row) {
     try {
       return delegate.extract(row);
     } catch (final Exception e) {
-      logger.error(timestampExtractErroMsg(e, row.toString()));
+      logger.error(RecordProcessingError.recordProcessingError(
```

Comment:
can we add a test case to `LoggingTimestampExtractorTest`?

Reply:
Certainly can - as its now easy with this test!